src/consumer.ts

```diff
@@ -389,8 +391,14 @@ export class Consumer extends EventEmitter {
             } else {
                 err.message = `Unexpected message handler failure: ${err.message}`
             }
+            const [message] = messages
```

Comment:
This is weird and I don't think it is very clear to everyone what does it do... I would at least do const [message,] = messages

Reply:
what? this is ES6 array destructuring

Reply:
furt hnusny

Reply:
🙄

Reply:
🥸